go.mod

```diff
@@ -3,7 +3,7 @@ module github.com/devfile/library
 go 1.13
 
 require (
-	github.com/devfile/api/v2 v2.0.0-20210713153530-f78ab9de1a30
+	github.com/devfile/api/v2 v2.0.0-20210804191700-4f7b6856e70a
```

Comment:
I did run the script but the schemas were not updated because I didn't make any changes there.  I just updated some of the tests and utils but thanks for bringing this to my attention for future consideration.  I thought the script only updated the dependencies and versions.